package.json

```diff
@@ -1,58 +1,89 @@
 {
-  "url": "https://github.com/mozilla/remote-newtab/issues",
-  "license": "MPL",
-  "contributors": [
-    {
-      "name": "Marcos Caceres"
-    },
-    {
-      "name": "Marina Samuel"
-    },
-    {
-      "name": "Olivier Yiptong"
-    },
-    {
-      "name": "Ursula Sarracini"
-    }
-  ],
-  "repository": "https://github.com/mozilla/remote-newtab",
+  "name": "newtab",
+  "version": "0.0.0",
+  "description": "Remote new tab (third version)",
+  "main": "src/main.js",
+  "scripts": {
+    "start": "npm run clean && npm run watch:js & npm run watch:css",
+    "build": "npm run clean && npm run build:js & npm run build:css",
+    "test": "karma start",
+    "test:watch": "karma start --no-single-run --browsers ChromeCanary",
+    "clean": "rm -rf ./www",
+    "build:js": "NODE_ENV=production webpack -p --optimize-dedupe",
+    "build:css": "node-sass src/main.scss -o www",
+    "watch:css": "npm run build:css && npm run build:css -- --source-map www/main.css.map -w -r",
+    "watch:js": "webpack-dev-server --hot --port=1944 --output-public-path www",
+    "generate-offline-files": "bin/generate-offline-files.js",
+    "karma": "karma start --single-run",
+    "test": "npm run karma",
+    "jshint": "jshint src bin",
+    "jscs": "jscs --esnext src",
+    "jscs:fix": "jscs --esnext --fix bin/ src/locale/ src/js/ src/js/lib/ src/test/ src/test/lib",
+    "pretest": "npm run generate-offline-files && npm run jshint && npm run jscs",
+    "posttest": "awk 'FNR==1{print \"\"}1' ./logs/reports/coverage/text-summary.txt ./logs/reports/coverage/text.txt"
```

Comment:
I think it would be better either remove or rename `pretest` and `posttest` or remove them and file followup tickets to restore coverage reporting/jscs